libs/sdk-backend-spi/src/workspace/execution/results.ts

```diff
@@ -77,85 +109,104 @@ export interface IMeasureDescriptor {
  */
 export interface ITotalDescriptor {
     // TODO: rename this to totalDescriptor ... the goal is to get rid of the overused 'header' nomenclature
-    totalHeaderItem: {
-        name: string;
-    };
+    totalHeaderItem: ITotalDescriptorItem;
+}
+
+/**
+ * Describes attributes to which the display form belongs.
+ *
+ * @public
+ */
+export interface IAttributeHeaderForm {
```

Comment:
I would use `IAttributeHeaderFormOf`: this is not some form, but rather specification of which attribute this header display form is of. So maybe even `IAttributeHeaderAttribute`?